covid_graphs/covid_graphs/country_graph.py

```diff
@@ -92,18 +97,32 @@ def adjust_xlabel(date: datetime.date):
             else:
                 return (date - self.log_xaxis_date_since).days
 
-        colors = ["SteelBlue", "Purple", "Green", "Orange", "Gray"][: len(self.traces)]
+        # Matplotlib colors.
+        color_by_event = {
+            BK_20200329: "rgb(255, 123, 37)",
+            BK_20200411: "rgb(0, 121, 177)",
+        }
+        extra_colors = [
+            "rgba(43, 161, 59, 0.4)",
```

Comment:
This color is too close to our light green background, we should probably increase opacity.

Reply:
We have to change the light green background, it does not make sense with multiple predictions.

Since we don't use the color, I'd leave that for @rejdi, the colors will adjust with the new frontend anyway I believe.

Reply:
I don't think it needs to be dropped entirely. The shade of green can change, each shaded area will correspond to another prediction date.